src/Resources/contao/library/Contao/Encryption.php

```diff
@@ -157,7 +166,12 @@ protected static function initialize()
 			throw new \Exception('Error initializing encryption module');
 		}
 
-		if (\Config::get('encryptionKey') == '')
+		/** @var KernelInterface $kernel */
+		global $kernel;
+
+		$container = $kernel->getContainer();
+
+		if ($container->getParameter('framework.secret') == '')
```

Comment:
I'm not sure, if this check is still needed - Symfony will raise a `ParameterNotFoundException` when `secret` is empty...